sklearn/feature_extraction/hashing.py

```diff
@@ -123,6 +128,11 @@ def fit(self, X=None, y=None):
         """
         # repeat input validation for grid search (which calls set_params)
         self._validate_params(self.n_features, self.input_type)
+
+        # optional if random_state is left to an integer seed
```

Comment:
`optional` is weird since there is nothing preventing the process of checking even if this is an integer.
Shall we just remove the comment then.